www/src/main.css

```diff
@@ -66,21 +69,27 @@ code {
 }
 
 .experimental {
-  margin: 3em 0;
-  padding: 1em;
-  border: 4px dashed red;
+  margin-top: 2rem;
+  padding: 1rem;
+  border: 0.2rem dashed red;
+
   color: red;
   font-weight: bold;
 }
 
+.link-centered {
+  display: block;
+  width: fit-content;
+  margin: 0 auto;
+  text-align: center;
+}
```

Comment:
This looks better on mobile but weird on desktop. I think I'd drop this suggestion.